saleor/dashboard/order/forms.py

```diff
@@ -290,6 +290,17 @@ class OrderNoteForm(forms.Form):
     )
 
 
+class OrderCustomerNoteForm(forms.ModelForm):
+    class Meta:
+        model = Order
+        fields = ["customer_note"]
+
+    customer_note = forms.CharField(
+        label=pgettext_lazy("Order customer note", "Customer note"),
```

Comment:
The context is really not helpful for the translators. (they only see two things: the context and the string to translate). The current context seems like we are displaying a customer note, not editing a field in a form.

```suggestion
        label=pgettext_lazy("Field for updating the customer's note on an order", "Customer note"),
```